sdk/textanalytics/azure-cs-textanalytics/src/main/java/com/azure/cs/textanalytics/TextAnalyticsAsyncClient.java

```diff
@@ -275,8 +275,10 @@ public TextAnalyticsServiceVersion getServiceVersion() {
      * @return A {@link DetectLanguageResult} equivalent for the error-ed document.
      */
     private DetectLanguageResult convertToErrorDetectLanguageResult(final DocumentError errorDocument) {
-        return new DetectLanguageResult(errorDocument.getId(), null, errorDocument.getError(),
-            null, null);
+        com.azure.cs.textanalytics.implementation.models.Error serviceError = errorDocument.getError();
```

Comment:
```suggestion
        Error serviceError = errorDocument.getError();
```

Reply:
As we have two models with the same name, we will have to use this to distinguish them